source/Keystone/ApplicationCredentials.rst

```diff
@@ -21,7 +24,7 @@ The form has 7 fields:
 
 - **Description:** Description of the application credential. This could be the description of which application will use this credential or the purpose of the credential.
 
-- **Secret:** A password for the credential which will be used in order to authenticate access through keystone. **This must be different to the user's password.**
+- **Secret:** A password for the credential which will be used in order to authenticate access through keystone. If left blank a secret will be generated automatically.
```

Comment:
`We recommend you leave this blank or use a tool like *pwgen* to generate this automatically`